action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -541,6 +556,19 @@ func (stateDB *StateDBAdapter) RevertToSnapshot(snapshot int) {
 			}
 		}
 	}
+	// restore access list
+	stateDB.accessList = nil
+	stateDB.accessList = stateDB.accessListSnapshot[snapshot]
+	{
```

Comment:
what's this bracket for?

Reply:
better for visual clarity. other places are in a code block by `if stateDB.fixSnapshotOrder`, here no need for the `if`, but leave the bracket to show the same code block